schema/python-impl.md

```diff
@@ -0,0 +1,85 @@
+# JSON Schema in Python
+
+Schema validation and enforcement is a great way to make for compatibles APIs that share a common interchange format.
+The existing tooling around JSON schemas is still in it's infancy, the [existing implementations](http://json-schema.org/implementations.html) are too much bound to the implementation details and lack 
+flexibility of high level abstraction, so that we optimally write the Python code with data members annotated pretty much like in `Go`  and utilize code & documentation generation to provide both the schemas specification and documentation.
+Furthermore, if we carefully decouple the operations, so that we first generate the schema, and then use the schema to generate the documentation we make half of the work *language invariant*. First step on the documentation front would be implementing sensible *Markdown* serializer for the schemas, so we can track them with GitHub.
+
+## Go-like annotations for Python
```

Comment:
@TomasTomecek  Is there any plan for Python 2.X support for type hints, or is it strictly 3.5+ ? Type declarations are nice, but rather inconclusive, as we need to attach more *state* than the type itself (textual description, required Y/N, value ranges etc.) - this could be possibly handled via meta-types but I guess that would be too much type system abuse and we could just write it in C# or C++ in the first place :)